R/colour-text.r

```diff
@@ -1,8 +1,9 @@
-
+# fix the check NOTE:Namespace in Imports field not imported from: ‘crayon’
```

Comment:
I do not understand why it is spurious: it complains about nothing being imported from a package listed in Imports.